hack/scale_out_poc/two_TP/nginx.conf

```diff
@@ -0,0 +1,128 @@
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+    worker_connections 768;
+    # multi_accept on;
+}
+
+http {
+
+    ##
+    # Basic Settings
+    ##
+
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+    types_hash_max_size 2048; 
+    proxy_http_version 1.1; 
```

Comment:
just realized. this is missing 

    proxy_read_timeout     1200;
    proxy_connect_timeout  1200;